tests/clone_test.go

```diff
@@ -412,6 +414,30 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 				expectEqualTemplateAnnotations(targetVM, sourceVM, key2)
 			})
 
+			It("clone and set hostname", func() {
+
+				sourceVM = createVM()
+
+				vmClone = generateCloneFromVMWithParams(sourceVM, targetVMName, targetHostname)
+				createCloneAndWaitForFinish(vmClone)
+
+				By(fmt.Sprintf("Getting the target VM %s", targetVMName))
+				targetVM, err = virtClient.VirtualMachine(sourceVM.Namespace).Get(context.Background(), targetVMName, &v1.GetOptions{})
```

Comment:
VirtualMachine doesn't use pointers anymore:
```suggestion
				targetVM, err = virtClient.VirtualMachine(sourceVM.Namespace).Get(context.Background(), targetVMName, v1.GetOptions{})
```